releasenotes/notes/0.19/7274-6f57628a7995a461.yaml

```
@@ -0,0 +1,11 @@
+---
+upgrade:
+  - |
+    In this version, ``from qiskit import *`` will not import submodules, but
```

This just reads like it's only for 0.19.0, maybe something like:

```suggestion
    Starting with this version, ``from qiskit import *`` will not import submodules, but
```

---

Done in 33ffd86f4.